runtimes/weave/src/lib.rs

```diff
@@ -885,6 +885,8 @@ impl pallet_chain_space::Config for Runtime {
 parameter_types! {
 	pub const MaxDigestsPerBatch: u16 = 1_000;
 	pub const MaxRemoveEntries: u16 = 1_000;
+	pub const MaxSelectiveDataKeyLength: u32 = 100;
```

Comment:
Same here, 64 would be good is what I think. and if its 64, should we keep this as u32 or u16/u8?

Reply:
Technically u8/u16 is enough. But some traits are not implemented for those types which is used in storage. So have used u32. Have to check.